Sources/Testing/Attachments/Attachment.swift

```diff
@@ -111,11 +111,9 @@ extension Attachment where AttachableValue == AnyAttachable {
 /// events of kind ``Event/Kind/valueAttached(_:)``. Test tools authors who use
 /// `@_spi(ForToolsIntegrationOnly)` will see instances of this type when
 /// handling those events.
-///
```

Comment:
Those are the ones which are bugging me the most! 😅 Since they cause warning diagnostics when building the docs.

Are you able to reference the tracking issue for that here? I'd like to assess how soon we anticipate it'll be fixed, and might want to still proceed with this in the short-term

Reply:
I'm thinking about https://github.com/swiftlang/swift-docc/pull/1107 in particular—the DocC team might be able to shoehorn this in at the same time if we ask nicely.

Reply:
Possible compromise:

```swift
// @Comment {
//   Swift's type system requires that this type be at least as visible as
//   `Event.Kind.valueAttached(_:)`, otherwise it would be declared private.
// }
```

i.e. keep `@Comment` in while switching from documentation comment to regular comment. I have a mild worry somebody will come along and think the double slashes are a typo and add these comments into the documentation block when we don't want them there.

Reply:
Done

Reply:
Yes, https://github.com/swiftlang/swift-docc/pull/1107 adds support for using `@Comment` in documentation comments. I'm planning on cherry picking it into 6.1

Reply:
Great to hear @franklinsch! I'll be happy to revert these specific portions once that lands in the toolchains we're using